### PR TITLE
Offline page not working in the Beez3 template

### PR DESCRIPTION
**Pull request by RickR2H**
Pull Request for Issue #15614 .

### Summary of Changes
The Beez3 template doesn't have an error.php so the default system template error.php is loaded. In that case the paths to the CSS files are incorrect as they are looking for the CSS files in the Beez3 template CSS folder and not in the system CSS folder.

### Testing Instructions

- Login in the Administrator
- Go to the templates and set the Beez3 template as default
- Go to Global Configuration and set the site offline
- Visit the website
- Now the login box has no styling because the stylesheets are missing
- Apply the patch
- The stylesheets for the login box is now loaded and the styling should be normal.

beez3-offline-page-patch

### Expected result
The login box should be centered and have styling

### Actual result
The login box has no styling because the stylesheets are not loaded

### CODE REVIEW

File: templates/system/offline.php [hunk 1]
```diff
@@ -20,14 +20,14 @@
 JHtml::_('script', 'jui/html5.js', array('version' => 'auto', 'relative' => true, 'conditional' => 'lt IE 9'));
 
 // Styles
-JHtml::_('stylesheet', 'offline.css', array('version' => 'auto', 'relative' => true));
+JHtml::_('stylesheet', 'templates/system/css/offline.css', array('version' => 'auto', 'relative' => false));
```
Comment (sanderpotjer) [inline templates/system/offline.php]: `relative` defaults to false, so you can simply leave `'relative' => false` out of the code here. So:

`JHtml::_('stylesheet', 'templates/system/css/offline.css', array('version' => 'auto'));`

Same for others below.

Comment (RickR2H) [inline templates/system/offline.php]: @sanderpotjer I Agree... It's a cleaner solution then! I'll change the code.

**Comment by sanderpotjer** [comment]
I have tested this item :white_check_mark: successfully on e5e516265f106ef3535f29a0864e90d5d6ec7da9<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/15711">issues.joomla.org/tracker/joomla-cms/15711</a>.</sub>

**Comment by franz-wohlkoenig** [comment]
I have tested this item :white_check_mark: successfully on e5e516265f106ef3535f29a0864e90d5d6ec7da9<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/15711">issues.joomla.org/tracker/joomla-cms/15711</a>.</sub>

**Comment by franz-wohlkoenig** [comment]
RTC after two successful tests.

**Comment by franz-wohlkoenig** [comment]
RTC after two successful tests.

**Comment by franz-wohlkoenig** [comment]
RTC after two successful tests.

**Comment by franz-wohlkoenig** [comment]
RTC after two successful tests.

**Comment by brianteeman** [comment]
Surely the changes in this PR are just masking a different problem

**Pull request merged**